src/scenes/NftDetailPage/NftDetailAsset.tsx

```diff
@@ -66,7 +67,15 @@ function NftDetailAssetComponent({ tokenRef, maxHeight }: NftDetailAssetComponen
     case 'AudioMedia':
       return <NftDetailAudio tokenRef={token.token} />;
     case 'ImageMedia':
-      return <NftDetailImage tokenRef={token.token} maxHeight={maxHeight} />;
+      return (
+        <NftDetailImage
+          tokenRef={token.token}
+          maxHeight={maxHeight}
+          // @ts-expect-error: we know contentRenderURL is present within the media field
+          // if token type is `ImageMedia`
+          onClick={() => window.open(token.token.media.contentRenderURL)}
```

Comment:
this was the result of an issue we face where even after filtering for `ImageMedia` token media types, typescript doesn't recognize that `contentRenderURL` is an available field